ops/dev/Dockerfile

```diff
@@ -26,7 +26,7 @@ RUN apt-get update && apt-get install -y \
 # Configure ssh server
 RUN mkdir /var/run/sshd
 RUN echo 'root:tba' | chpasswd
-RUN sed -i 's/PermitRootLogin prohibit-password/PermitRootLogin yes/' /etc/ssh/sshd_config
+RUN sed -i 's/#PermitRootLogin prohibit-password/PermitRootLogin yes/' /etc/ssh/sshd_config
```

Comment:
This is commented out by default in the new config - we could also replace this line with

```
RUN echo "PermitRootLogin yes" >> /etc/ssh/sshd_config
```

Either way works - I kept the `sed` mostly to show that removing the `sed` in favor of an `echo` wouldn't be a regression